### ISSUE TITLE
Changes to {N} templates do not update

### ISSUE THREAD

**Issue by barryajones** [nstudio/xplat#45]
I've created a new xplat nativescript project using the guide:

```
npm install -g @nrwl/schematics
npm install -g @angular/cli
create-nx-workspace test-app
```

```
npm install @nstudio/schematics --save-dev
```

```
ng g xplat --prefix=foo --platforms=web,nativescript
```

```
ng g app.nativescript appname
npm i
```

The app compiles successfully when I run `npm run start.nativescript.appname.android` and I can see the app screen with the text `hello xplat`.

If I try to make a change to either the en.json locale or the **Label** element in app.component.ts to something else I see webpack pick up on the change and it recompiles. However the label does not update in the android emulator. 

```html
<Label automationText="helloLabel" text="Hello World" class="h1 p-10 text-center"></Label>
```

I've tried stopping the npm script and starting it again but I still see the old label. I have to run `npm run clean.nativescript.app` and then `npm run start.nativescript.app.android` to see the change.

I also have a another xplat project that has both a web and nativescript app and I see the same issue with the nativescript app. I have no problems with the web app updating.

I uploaded the demo to a github repo: https://github.com/barryajones/xplat-demo

**Comment by NathanWalker** [comment]
What version of `node`, `npm` and nativescript do you have installed on your system.
Also run `tns info` and post what you have here.

**Comment by barryajones** [comment]
Thanks for the quick reply @NathanWalker.

**Node** `v8.9.4`
**npm** `5.6.0`
**NativeScript** `nativescript@4.1.2`

```
$ tns info
✔ Getting NativeScript components versions information...
⚠ Update available for component nativescript. Your current version is 4.1.2 and the latest available version is 5.0.0.
```

I updated nativescript to version 5 and it's now working! :)

```
npm i -g nativescript
```
